vowpalwabbit/distributionally_robust.cc

```diff
@@ -0,0 +1,176 @@
+#include <cmath>
+#include <list>
+#include <tuple>
+#include <type_traits>
+
+#include "distributionally_robust.h"
+
+namespace VW
+{
+namespace distributionally_robust
+{
+
+double ChiSquared::chisq_onedof_isf(double alpha)
```

Comment:
Can you mark this function as VW_STD14_CONSTEXPR?

Reply:
constexpr function 'VW::distributionally_robust::ChiSquared::chisq_onedof_isf' cannot result in a constant expression

Reply:
Maybe because its a member function? Not important though, leave it out